ui/page/main_page.go

```diff
@@ -229,6 +226,7 @@ func (mp *MainPage) setLanguageSetting() {
 }
 
 func (mp *MainPage) UpdateBalance() {
+	go load.GetUSDExchangeValue(&mp.dcrUsdtBittrex)
```

Comment:
the `UpdateBalance` looks to be called multiple times, which will start up multiple go routines... not good. 
You can simply add a condition to check if the usd values is available. if it is not, the fetch it and refresh window. if it is, skip fetching it.

Issue i see here is that if the USD exchange val is updated, the total bal will still be same as `UpdateBalance` is called when there is a new block and tx from the listener.go file.

You would need a way to track and ensure that the USD exchange value is updated with each new block and tx.